realionetwork_3300-4/genesis.json

```diff
@@ -0,0 +1,615 @@
+{
+    "app_name": "realio-networkd",
+    "app_version": "1.0.2-2-gd363425",
+    "genesis_time": "2025-01-23T17:37:49.467749Z",
+    "chain_id": "realionetwork_3300-4",
+    "initial_height": 1,
+    "app_hash": null,
+    "app_state": {
+      "07-tendermint": null,
+      "asset": {
+        "params": {},
+        "tokens": [
+          {
+            "authorizationRequired": true,
+            "authorized": [
+              {
+                "address": "realio1rj6my433u6wzzev6xvduggwvq30pfhwtsac3qa",
+                "authorized": true
+              },
+              {
+                "address": "realio1mx3h3zue85jj0u48uv9eyr3yprkqr70d0x0egr",
```

Comment:
lets add in LMX also:

`{
          "authorizationRequired": true,
          "authorized": [
            {
              "address": "realio1axjfxxdr7jdgtmd6s6cgsk82280xqnnwv92uc8",
              "authorized": true
            }
          ],
          "manager": "realio1axjfxxdr7jdgtmd6s6cgsk82280xqnnwv92uc8",
          "name": "LMX Token",
          "symbol": "lmx",
          "total": "100000000"
        }`